latex/epoch.tex

```diff
@@ -289,6 +394,28 @@
   \label{fig:ts-types:pool-clean}
 \end{figure}
 
+We now present the pool-cleanup transition rule in Figure~\ref{fig:rules:pool-clean}.
+This rule will be applied whenever there is one or more stake pools scheduled
+to retire this epoch. If so, all of the entries in $\var{stpools}$,
+$\var{pparams}$, and $\var{retiring}$ which correspond to any of the hash keys
+of the stake pools scheduled to retire this epoch are removed from
+these variables.
+
+In addition, the $\var{deposits}$ value on the ledger is
+updated by subtracting from the existing $\var{deposits}$ value the total
+value of all (decayed) refunds for the stake pools scheduled for retirement
+this epoch. The refunds for stake pool retirement end up in the rewards
+accounts of the pool key holders. Specifically, the set of rewards is updated by
+adding to coin value corresponding to each key entry in the $\var{rewards}$
+finite map the amount of (decayed) refund intended for the pool
+key which deregisters the pool.
+
+\begin{todo}
+  So, refunds become rewards?
+
+  What happens if the indivudual key is deregistered?
```

Comment:
I think we decided no not allow this. So we now need a check in the key deregistration that the key in question is not a registered stake pool. It may also make sense to move the value inside a reward address into a UTxO when a key is deregistered.